modules/aws/awc/main.tf

```diff
@@ -110,6 +118,7 @@ data "template_file" "user-data" {
   vars = {
     bucket_name         = var.bucket_name,
     provisioning_script = local.provisioning_script,
+    log_manager_script  = var.log_manager_script,
```

Comment:
_Just a cross reference mention here to comment about mixed `local`/`var` usage on line [13](https://github.com/EditShareCo/cloud_deployment_scripts_hp_inc/pull/5/files#diff-51d281d110c60958957a4a5e352dd6e2aa0aa02de13680b3df9564a0dfbe6852R13)_